Update to avoid nil context

balexandr opened the pull request:
Update logic to use `&` as the logic's lambda gets set to [] in the console which could cause confusion. We noticed in the `ConsoleFilterToggles` class that this `Rails.application.config.filter_parameters` gets overridden to `[]` when loading up the rails console `if defined?(Rails::Console)`. So if you were to test-run a Sidekiq job or something it would throw an error:

<img width="2684" height="490" alt="image" src="https://github.com/user-attachments/assets/18977610-d0de-4ec2-88fd-87ecba659196" />

This fix is just to ensure it doesn't throw an error when running anything in the terminal


## Related issue(s)

- Not ticket, noticed it while testing something else

balexandr commented:
Not the fix

rmtolmach commented:
@balexandr can you update the description and add a related issue if you have one? Can you also show a before/after testing output to demonstrate what this fixes?

balexandr commented:
> @balexandr can you update the description and add a related issue if you have one? Can you also show a before/after testing output to demonstrate what this fixes?

Added a screen shot of what was happening and updated desc
